routes/renderers.js

```diff
@@ -189,10 +177,6 @@ function _render(req, res, next, opts) {
             }
         })
         .catch(function failure(err) {
-            // If path on disk does not exist at all, return 404 not found.
-            if (err.code === 'ENOENT') {
-                err.status = 404;
-            }
```

Comment:
See `notebook-fs.js` line 40. It is part of an effort to pull `fs` error code checking into the modules actually performing the filesystem operations. The HTTP error status can be set by the storage modules. This will help make it easier to swap the storage modules in the future if we want to switch from a filesystem store to some other storage solution.